ethers-providers/src/transports/quorum.rs

```diff
@@ -277,23 +277,20 @@ pub enum Quorum {
 }
 
 impl Quorum {
-    fn weight<'a, T: 'a>(
-        self,
-        providers: impl IntoIterator<Item = &'a WeightedProvider<T>>,
-    ) -> u64 {
+    fn weight<T>(self, providers: &[WeightedProvider<T>]) -> u64 {
         match self {
-            Quorum::All => providers.into_iter().map(|p| p.weight).sum::<u64>(),
+            Quorum::All => providers.iter().map(|p| p.weight).sum::<u64>(),
```

Comment:
iirc this was previously `iter()` and then a previous PR changed to `into_iter()` - wondering why the change back?

Reply:
Because I undid changes in an earlier commit and this was how they had it written.